db/schema.rb

```diff
@@ -59,10 +61,6 @@
     t.index ["user_id"], name: "index_posts_on_user_id"
   end
 
-  create_table "students", id: :serial, force: :cascade do |t|
```

Comment:
Good question! It popped up on the master branch at some point, not sure when it was created